flexget/plugins/clients/transmission.py

```diff
@@ -682,7 +685,7 @@ def on_task_exit(self, task, config):
                                 is_minratio_reached or
                                 (is_torrent_seed_only and is_torrent_idlelimit_since_added_reached) or
                                 (not is_torrent_seed_only and is_torrent_idlelimit_since_finished_reached))
-                    and is_tracker_matching and is_directories_matching):
+                    and is_directories_matching (not is_preserve_tracker_matching and is_tracker_matching)):
```

Comment:
>  and is_directories_matching (not is_preserve_tracker_matching and is_tracker_matching)

is there meant to be an 'and' or an 'or' in between is_directories_matching and that parenthetical?